### PR TITLE
oopsy: add emerald weapon ex warnings

### PR DESCRIPTION
**Pull request by quisquous**

### CODE REVIEW

**Review by panicstevenson — APPROVED** [review]

**Review by JLGarber — APPROVED** [review]

**Comment by quisquous** [comment]
I do want to merge this, but I kind of wanted to try this out at least once, since there's a bunch of complicated orb warnings that are totally untested.

If anybody happens to try this out, please let me know!

**Comment by panicstevenson** [comment]
I'll be able to test this tonight.

File: ui/oopsyraidsy/data/05-shb/trial/emerald_weapon-ex.js
Comment (panicstevenson) [inline ui/oopsyraidsy/data/05-shb/trial/emerald_weapon-ex.js:17]:
```suggestion
    'EmeraldEx Sidescathe 2': '55D5', // left/right cleave
```

File: ui/oopsyraidsy/data/05-shb/trial/emerald_weapon-ex.js
Comment (Jaehyuk-Lee) [inline ui/oopsyraidsy/data/05-shb/trial/emerald_weapon-ex.js:44]:
```suggestion
            fr: `${matches.ability} (seul(e))`,
            ko: `${matches.ability} (혼자 맞음)`,
```

File: ui/oopsyraidsy/data/05-shb/trial/emerald_weapon-ex.js
Comment (Jaehyuk-Lee) [inline ui/oopsyraidsy/data/05-shb/trial/emerald_weapon-ex.js:92]:
```suggestion
            fr: `${matches.ability} (mauvaise couleur)`,
            ko: `${matches.ability} (틀린 색깔)`,
```

File: ui/oopsyraidsy/data/05-shb/trial/emerald_weapon-ex.js
Comment (Jaehyuk-Lee) [inline ui/oopsyraidsy/data/05-shb/trial/emerald_weapon-ex.js:107]:
```suggestion
            fr: `${matches.ability} (mauvaise couleur)`,
            ko: `${matches.ability} (틀린 색깔)`,
```

**Review by Jaehyuk-Lee — COMMENTED** [review]
Korean translations

File: ui/oopsyraidsy/data/05-shb/trial/emerald_weapon-ex.js
Comment (quisquous) [reply ui/oopsyraidsy/data/05-shb/trial/emerald_weapon-ex.js]: Sorry, I've removed this from the PR because I haven't been back in emerald and I don't think these are working.  I think in general it's better to wait until after PRs have landed to add translations.  I should have said the same to Akurosia before merging in (and then removing) those translations.

**Pull request merged**
